azure-wildfly/src/test/resources/sunstone.properties

```diff
@@ -13,4 +13,12 @@ sunstone.wildfly.mgmt.connection.timeout=120000
 sunstone.wildfly.mgmt.host=master
 sunstone.wildfly.mgmt.profile=default
 
+# valid for marketplace-image profile
+ts.azure.eap.image.plan.name=${azure.image.plan.name}
```

Comment:
Perhaps we can set some defaults here. WDYT?

Reply:
good idea